lms/djangoapps/program_enrollments/rest_api/v1/views.py

```diff
@@ -911,14 +912,18 @@ def get(self, request, program_uuid=None):
 
             certificate_info = get_certificate_for_user(user.username, enrollment.course_id) or {}
 
+            due_dates = []
+            if overview.has_started() and not overview.has_ended():
```

Comment:
can you just check the value of `get_course_run_status` in order to avoid duplicating this logic?

Reply:
oh yeah, didn't even notice that function. I'll use that instead